### ISSUE TITLE
It would be very useful to be able to run the code even if mayavi and vtk are not installed

### ISSUE THREAD

**Issue by dccote** [DCC-Lab/PyTissueOptics#117]
**Is your feature request related to a problem? Please describe.**
The code may need to run on headless servers that may not have mayavi or vtk and it may be very difficult to install them. It would be nice to separate the structures of `scene` from its visualization.

**Describe the solution you'd like**
Be able to run the code without visualisation tools installed.

**Comment by JLBegin** [comment]
I agree that we should have headless support, but I doubt it should be mayavi/3D specific.
